### PR TITLE
deploy v1

### PR DESCRIPTION
**Pull request by spengrah**
Deploy v1 to the following networks:

- ethereum (mainnet)
- polygon
- gnosis chain
- arbitrum
- optimism
- goerli (testnet)
- sepolia (testnet)

### CODE REVIEW

File: script/Hats.s.sol [hunk 1]
```diff
@@ -7,32 +7,41 @@ import { Hats } from "../src/Hats.sol";
...
-        console2.log(vm.getNonce(deployer));
+        console2.log("Deployer: ", deployer);
+        console2.log("Deployer Nonce: ", vm.getNonce(deployer));
 
         vm.startBroadcast(deployer);
 
+        // deploy Hats
         Hats hats = new Hats{ salt: SALT }(name, baseImageURI);
 
+        // mint Hats Protocol Governance tophat
```
Comment (nintynick) [inline script/Hats.s.sol:26]: I would note that this is not connected to any authorities and that the v1 protocol is permissionless and not upgradable

**Review by nintynick — APPROVED** [review]
🔥

File: script/Hats.s.sol (hunk 1 above)
Comment (spengrah) [reply script/Hats.s.sol:26]: I'm planning to note this in Releases, but can also note here if useful. wdyt @nintynick?

Comment (spengrah) [reply script/Hats.s.sol]: updated in https://github.com/Hats-Protocol/hats-protocol/pull/119/commits/8f1aae4c21c723c84e8e76bc6393f45f2f8cd347

**Pull request merged**